### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Rasmusafj**
Merges development branch into master, so the cleaning process of the repo can start correctly.

### CODE REVIEW

File: danspeech/DanSpeechRecognizer.py [hunk 1]
```diff
@@ -12,6 +13,7 @@ def __init__(self, model_name=None, lm_name=None,
                  beam_width=64):
 
         self.device = torch.device("cuda" if with_gpu else "cpu")
+        print(self.device)
```
Comment (zer0x42) [inline danspeech/DanSpeechRecognizer.py:16]: printing a plain number isn't all that informative, consider adding a meaningful text as well.

File: danspeech/DanSpeechRecognizer.py [hunk 2]
```diff
@@ -39,10 +41,28 @@ def __init__(self, model_name=None, lm_name=None,
...
+
+        self.second_model = DanSpeechPrimary()
+        self.audio_config = self.second_model.audio_conf
+        self.second_model =  self.second_model.to(self.device)
+        self.second_model.eval()
+
+        self.second_decoder = DSL3gram()
+
+        # When updating model, always update decoder because of labels
+        self.second_decoder = decoder = BeamCTCDecoder(labels=self.second_model.labels, lm_path=self.second_decoder,
```
Comment (zer0x42) [inline danspeech/DanSpeechRecognizer.py:57]: I think we should add an exception here for when BeamCTC isn't installed, I'm guessing that will happen rather often.

File: danspeech/DanSpeechRecognizer.py [hunk 3]
```diff
@@ -87,10 +107,80 @@ def update_decoder(self, lm=None, alpha=None, beta=None, labels=None, beam_width
...
+        self.iterating_transcript = ""
+        self.full_output = []
+        self.spectrograms = []
+
+    def streaming_transcribe(self, recording, is_last, is_first):
+        recording = self.audio_parser.parse_audio(recording, is_last)
+
+        transcript = ""
+        if len(recording) != 0:
+            # ToDO: Remove but keep here for now
```
Comment (zer0x42) [inline danspeech/DanSpeechRecognizer.py:127]: extend explanation, what is affected by the toDo, the entire assertion?

Comment (Rasmusafj) [reply danspeech/DanSpeechRecognizer.py:127]: Will also be cleaned! :)

File: danspeech/DanSpeechRecognizer.py (hunk 2 above)
Comment (Rasmusafj) [reply danspeech/DanSpeechRecognizer.py:57]: Agree, it will be refactored in upcoming PR! :)

File: danspeech/DanSpeechRecognizer.py (hunk 1 above)
Comment (Rasmusafj) [reply danspeech/DanSpeechRecognizer.py:16]: Will also be cleaned! :)

**Pull request merged**